skietbaan_fe/src/components/ScoreCapture.js

```diff
@@ -103,7 +105,7 @@ export default class search extends Component {
 
   Validate() {
     let Valid = false;
-    if (parseInt(this.state.score) < 1 || this.state.score === null 
+    if (parseInt(this.state.score,0) < 1 || this.state.score === null 
```

Comment:
You don't have to do this, some argue it's not needed but it will help you when debugging and trying to understand which statements are together and which comparisons are grouped together. 
Make use of parenthesis to create logical groupings eg:
```
    if ((parseInt(this.state.score,0) < 1)
	|| (this.state.score === null)
	|| ((this.state.score % 1) !== 0))
```

Reply:
fixed